pkg/atls/atls.go

```diff
@@ -226,39 +220,13 @@ func GetCertificate(caUrl string, cvmId string) func(*tls.ClientHelloInfo) (*tls
 				return nil, fmt.Errorf("failed to create CSR: %w", err)
 			}
 
-			csrData := string(csr.CSR)
-
-			r := csrReq{
-				CSR: csrData,
-			}
-
-			data, sdkErr := json.Marshal(r)
-			if sdkErr != nil {
-				return nil, fmt.Errorf("failed to marshal CSR request: %w", sdkErr)
-			}
-
 			notBefore := time.Now()
 			notAfter := time.Now().AddDate(notAfterYear, notAfterMonth, notAfterDay)
 			ttlString := notAfter.Sub(notBefore).String()
 
-			query := url.Values{}
-			query.Add("ttl", ttlString)
-			query_string := query.Encode()
-
-			certsEndpoint := "certs"
-			csrEndpoint := "csrs"
-			endpoint := fmt.Sprintf("%s/%s/%s", certsEndpoint, csrEndpoint, cvmId)
-
-			url := fmt.Sprintf("%s/%s?%s", caUrl, endpoint, query_string)
-
-			_, body, err := processRequest(http.MethodPost, url, data, nil, http.StatusOK)
+			cert, err := caSDK.IssueFromCSR(cvmId, ttlString, string(csr.CSR), domainId, "")
```

Comment:
which value is empty string

Reply:
also shouldn't this be IssueFromCSRInternal https://github.com/absmach/certs/pull/156#discussion_r2335856335

Reply:
also shouldn't this be IssueFromCSRInternal https://github.com/absmach/certs/pull/156#discussion_r2335856335

Reply:
also shouldn't this be IssueFromCSRInternal https://github.com/absmach/certs/pull/156#discussion_r2335856335